### PR TITLE
Weather app project

### PR DESCRIPTION
**Pull request by ricardomonte**
I built the weather app project as the fourth project of the JavaScript module from Microvers. Here you can look for the current weather of a city around the world.

The main goal was to be able to fetch an API in this case, OpenWeatherMap API, using Promes and .then or using async and await. In this project, I decided to use async and await.

Built with:

- HTML
- SCSS
- JavaScript
- Webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+[Live demo](https://amazing-golick-931359.netlify.app/)
+
+##  :construction_worker: Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+- Go to the main page.
+- Press the "Code" button and get the repo link.
+- Clone it using git command "git clone".
```
Comment (whiz25) [inline README.md:26]: - Give the exact command for cloning the repo
- Also add instructions for what needs to be done after cloning. How should one open the app?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open"
+  },
```
Comment (whiz25) [inline package.json:10]: - Scripts should look like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nicely done @ricardomonte . You took care of most requirements for this project. A user can view weather information for a given location, the background changes based on the weather value. Good!
- [ ] The toggle functionality of the app doesn't work as expected. Right now, one can only toggle between `C` and `F` when they are doing the search but not after. One should be able to toggle even after the results of the search are received.
- [ ] Read my comments for more suggestions

### In Your Next Projects
Please do the following in your next projects
- [ ] Do not commit `.env` or any file containing sensitive information such as your `API_KEY` as this can be used by someone for the wrong reasons. There's a text field for extra information such as API_KEYs when submitting projects for review. Please use that next time.
- [ ] Capitalize your commit messages. Instead of `add imges for backgrounds`, do `Add imges for backgrounds`
- [ ] The `assets` folder is meant for things like `images` not `javascript`. Javascript code should live in `src/js`



### Martin Afani

**Review by Forison — APPROVED** [review]
## Status::Approved✔️

### Good job on addressing the issues raised earlier the toggle effect is functional and the necessary build commands have been added🙂 .

> As mentioned earlier🙂 :
>  The assets folder is meant for things like images not javascript. Javascript code should live in src/js and Give the exact command for cloning the repo
Also add instructions for what needs to be done after cloning. How should one open the app? after build user should visit the dist folder to launch the application.

#### That is it, kindly go ahead and merge the branch and proceed to the next task👨‍🎓.

#### Happy coding👨‍💻
Addo Forison

**Comment by ricardomonte** [comment]
Hello, Addo Forison thanks for your comment I would like to ask you if you can send me where I can read about the assets folder. As far as I understand assets folder can contain images, CSS, and js. (in front-end)
thank you again.

**Pull request merged**